scripts/hail_batch/hgdp1kg_tobwgs_densify/hgdp_1kg_tob_wgs_densify.py

```diff
@@ -35,11 +36,40 @@ def query(output):  # pylint: disable=too-many-locals
         & hl.is_defined(tob_wgs.index_rows(hgdp_1kg['locus'], hgdp_1kg['alleles']))
     )
     tob_wgs = tob_wgs.semi_join_rows(hgdp_1kg.rows())
-    tob_wgs = tob_wgs.cache()
-    print(tob_wgs.count_rows())
     tob_wgs_path = f'{output}/tob_wgs_filtered.mt'
     tob_wgs = tob_wgs.repartition(1000, shuffle=False)
-    tob_wgs.write(tob_wgs_path)
+    if not hl.hadoop_exists(tob_wgs_path):
+        tob_wgs.write(tob_wgs_path)
+    tob_wgs = hl.read_matrix_table(tob_wgs_path)
```

Comment:
I think you're trying to shortcut the computation here, but I don't think this works as you expect, given that `hgdp_1kg` would still need to be computed.

It might be better to create a separate script where you filter the rows of `hgdp_1kg` based on the rows that are present in the MT you've read from `tob_wgs_path`.

Reply:
could I just start the script by reading in `hgdp1kg_tobwgs_joined_all_samples.mt` (the filtered file from the precious script), and then perform the PCA?

Reply:
sorry, I didn't look at the script correctly. I've filtered the hgdp/1kg data based off of your suggestions using the filtered tob_wgs matrix table